src/components/CardTable/CardTable.jsx

```diff
@@ -218,18 +236,22 @@ export default class CardTable extends Component {
   }
 
   updateTableNodeRef = (ref) => {
-    this.tableNodes = {
-      ...this.tableNodes,
-      [ref.id]: ref,
+    if (ref) {
```

Comment:
During dismounts it always passes in null. This normally wouldn't be much of an issue, except we try to pull out the id